documentation/examples/remote_storage/remote_storage_adapter/main.go

```diff
@@ -25,6 +25,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/blang/semver"
```

Comment:
I think it's something to make clear with the 1.7 release, we could time submission accordingly.

The other adapter in this repo also needs changing.